src/socket/socket.js

```diff
@@ -86,19 +84,59 @@ export function initializeSocket(httpServer) {
         return;
       }
       // Example usage of modifyPoints function
-     
+
       const leaderboard = generateLeaderboard(roomId);
       io.to(socket.id).emit("leaderboard", leaderboard);
     });
 
+    let imageUploadOrder = [];
+
+    // Modify the socket.on function to keep track of the image upload order
     socket.on("upload", (data) => {
       console.log("got an image");
       const base64Data = data.image.replace(/^data:image\/\w+;base64,/, "");
       const buffer = Buffer.from(base64Data, "base64");
       const filename = Date.now() + "-" + data.filename;
       const isCorrectObject = checkImage(filename, buffer, "bottle");
       console.log(isCorrectObject);
+
+      // Add the username to the image upload order
+      imageUploadOrder.push(data.username);
+
+      // Call modifyPoints function with the room ID and updated point modifiers
+      modifyPoints(data.roomId);
```

Comment:
as i explained earlier, this whole function could've been in roomManager.js file